src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -907,6 +907,65 @@ public override IOperation VisitWhileLoop(IWhileLoopOperation operation, int? ca
             return null;
         }
 
+        public override IOperation VisitVariableDeclarationGroup(IVariableDeclarationGroupOperation operation, int? captureIdForResult)
+        {
+            // We erase variable declarations from the control flow graph, as variable lifetime information is contained in a parallel data structure.
+            foreach (var declaration in operation.Declarations)
+            {
+                VisitVariableDeclaration(declaration, captureIdForResult);
+            }
+
+            return null;
+        }
+
+        public override IOperation VisitVariableDeclaration(IVariableDeclarationOperation operation, int? captureIdForResult)
+        {
+            foreach (var declarator in operation.Declarators)
+            {
+                ILocalSymbol localSymbol = declarator.Symbol;
+
+                IOperation initializer = null;
+                SyntaxNode assignmentSyntax = null;
+                if (declarator.Initializer != null)
+                {
+                    initializer = Visit(declarator.Initializer);
```

Comment:
>declarator.Initializer [](start = 40, length = 22)

We should skip through IVariableInitializerOperation to the Value here

Reply:
I'm planning on doing that skip in a separate PR for all the initializer types. I don't think the skip though should be here, the visitor for IVariableInitializer should just remove itself from the tree and return visit of it's child.

---
In reply to: [166775123](https://github.com/dotnet/roslyn/pull/24692#discussion_r166775123) [](ancestors = 166775123)

Reply:
It should be skipped here and in context of this PR. IVariableInitializerOperation doesn't represent a value and doesn't belong on the right hand side of the assignment. The remaining initializer types can wait as we haven't even decided what shape should the graph have for them and do not have an API for requesting it yet.

---
In reply to: [166777002](https://github.com/dotnet/roslyn/pull/24692#discussion_r166777002) [](ancestors = 166777002,166775123)